ome-dundeeomero.yml

```diff
@@ -228,6 +228,25 @@
         dest: /etc/cron.daily/nightly-pg_dump-omero.sh
         mode: "u=rwx,go="
 
+    - name: Create a figure scripts directory
+      become: yes
+      file:
+        path: /opt/omero/server/OMERO.server/lib/scripts/omero/figure_scripts
+        state: directory
+        mode: 0755
+        recurse: yes
+        owner: "omero-server"
```

Comment:
@pwalczysko - Thanks for this PR! This has been on my wish list for a while. In this case, the omero server O.S. user is unique to this host - see line :172.

Reply:
Does anyone know whether the user actually matters here, or could it be owned by root?

Reply:
@kennethgillen 
>omero server O.S user is unique....
Sorry, do not understand the context of this remark. What does the uniquenness imply ? What should I do about it ?

Reply:
@pwalczysko meaning that the user that owns OMERO differs across hosts. Nightshade uses a different name.

Reply:
@joshmoore : Thank you. And the name is.. ? The line 172 mentioned by @kennethgillen resolves to me as https://github.com/openmicroscopy/prod-playbooks/pull/10/files#diff-896819b5bdf43f197f838c7cc66babf4R172 - I know the context now, but not the answer :)

Reply:
@pwalczysko - line 172 shows an example of existing usage of that user within the context of this playbook.

Reply:
Mentioned to @pwalczysko was referring to `ome-dundeeomero.yml` rather than `ome-demoserver.yml`, and that my line :172 was a few lines out - was referring to :169 https://trello.com/c/1qSXnrPt/13-demo-541-upgrade#comment-5a266dcb5d638979135e3fe3